packages/base/room.gts

```diff
@@ -23,7 +23,11 @@ import {
   type CardRef,
 } from '@cardstack/runtime-common';
 
-const cardVersions = new Map<Card, string>();
```

Comment:
Honestly, I was not too sure about saving cardVersions with key but just went with it. I thought it would be card.id which wud be the key for versions

Reply:
Also perhaps there is a reason behind this, `cardVersions` seems to be a global variable which seems to share state across rooms. Perhaps, we literally want to store all versions inside this particular realm. I dono if this makes sense inside the base room card.